SUMMARY.md

```diff
@@ -40,6 +40,10 @@
   * [Professional Edition](products/editions/professional-edition.md)
   * [Enterprise Edition](products/editions/enterprise-edition/README.md)
     * [B2B Suite](products/editions/enterprise-edition/b2b-suite.md)
+    * [Concepts](products/editions/enterprise-edition/concepts/README.md)  
```

Comment:
These pages should be located inside the "B2B Suite" tree, not next to it.

Reply:
ah my fault 😅